script/run-docker

```diff
@@ -0,0 +1,106 @@
+#!/bin/sh
+
+# Source the `config/environment` script.
+#
+# TODO This should be removed?  Ideally, if the two environment
+# variables are set, we should be all good to go.
+. $(dirname $0)/../config/environment
+
+# Locate our docker binary.
+docker=$(which docker)
+
+# Start writing out the command.
+command="$docker run -it"
+
+if [ -z "${CS251TK_IMAGE}" ];
```

Comment:
And I see them down here, so… what about just implying some defaults in here, and the README says that you can change the values with these env vars?